pkgs/desktops/pantheon/desktop/gala/default.nix

```diff
@@ -78,25 +78,6 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./plugins-dir.patch
-    # Add systemd units so gnome-session can launch Pantheon
-    # using the systemd units instead of the XDG autostarts.
-    # https://github.com/elementary/gala/pull/1256
-    (fetchpatch {
-      url = "https://github.com/elementary/gala/commit/d425959aa2b9b9ac7ef622f601f3de1ca8ee768a.patch";
-      sha256 = "1r5jwmgvg970kz2yx79mz7cn6bfivsbqmi6s497q8z8lj8xrxr0g";
-    })
-    # autostarts: set hidden under systemd
-    # Prevents gnome-session from autostarting gala if systemd
-    # is launching the session instead.
-    # https://github.com/elementary/gala/pull/1255
-    (fetchpatch {
-      url = "https://github.com/elementary/gala/commit/bd362ec0c89bd72c5afca3f8313ee70d7ee25357.patch";
-      sha256 = "1l8dzyk7yncppg8yzhk7ddpccg3c0j95sar19hckc4wy4c1bn1j0";
-    })
-    # Hope this can make WITH_SYSTEMD block functions
-    # and make libsystemd found by pkg-config
-    # https://github.com/NixOS/nixpkgs/issues/101817#issuecomment-932956940
-    ./fix-gala-x11-timeout.patch
```

Comment:
The patch is accepted on upstream, however seems that I cannot change this part to `fetchpatch` as gala fail to build.

- https://github.com/NixOS/nixpkgs/issues/32084

Maybe I will just bump gala to a unstable version, there are only translation updates beside the systemd unit change.